environment.yml

```
@@ -0,0 +1,16 @@
+name: deltakit-conda
+channels:
+- conda-forge
+- nodefaults
+dependencies:
+- pandoc >=3.7.0.2,<4
+- python >=3.10
+- actionlint >=1.7
+- pixi-pycharm >=0.0.8,<0.0.9
```

```suggestion
- python >=3.10
```
wouldn't the other be installed at least with the `-e ./[dev]`?

---

No, as things are, none of these are installed with `-e ./[dev]`. You can tell because they are not listed in `pyproject.toml`.

We might be able to install `actionlint` via `actionlint-py`, but that is not what we currently have. 

We can remove `pixi-pycharm`, since that would not be needed for conda users, but the rest are needed for some development activities.

---

Done. Leaving `actionlint` and `pandoc` alone for now.